uwblueprint/ccbc
# Omar/update review service

## Notion ticket link

[Ticket ](https://www.notion.so/uwblueprintexecs/Edit-a-review-4454f563305046e0940842a8b3b08627)



## Implementation description
*  Added the update review service.



## Steps to test
1. Create a review using postman and then update it and verify that the field match as expected.



## What should reviewers focus on?
* 


## Checklist
